config/kubernetes/org.yaml

```
@@ -143,6 +143,7 @@ members:
 - butuzov
 - cadmuxe
 - caesarxuchao
+- camilamacedo86
 - calebamiles
```

nit: the list should be lexically sorted.

```suggestion
- calebamiles
- camilamacedo86
```